services/ui-src/src/components/banners/AdminBannerProvider.tsx

```diff
@@ -1,28 +1,35 @@
 import { useState, createContext, ReactNode, useMemo, useEffect } from "react";
 // utils
 import { AdminBannerData, AdminBannerShape } from "utils/types/types";
-import { bannerId } from "../../utils/constants/constants";
+import { bannerId, GET_BANNER_FAILED } from "../../utils/constants/constants";
+import { errorHandler } from "utils/errors/errorHandler";
 // api
 import { deleteBanner, getBanner, writeBanner } from "utils/api/index";
 
 const ADMIN_BANNER_ID = bannerId;
 
 export const AdminBannerContext = createContext<AdminBannerShape>({
   bannerData: {} as AdminBannerData,
-  fetchAdminBanner: () => {},
-  writeAdminBanner: () => {},
-  deleteAdminBanner: () => {},
+  fetchAdminBanner: Function,
+  writeAdminBanner: Function,
+  deleteAdminBanner: Function,
+  errorData: null,
```

Comment:
Added `errorData` to banner context so errors during banner fetch can be displayed downstream.